java-frontend/src/test/java/org/sonar/java/se/JavaCheckVerifier.java

```diff
@@ -336,7 +332,7 @@ private void validateFlowMessages(List<AnalyzerMessage> actual, String flowId, C
       .forEach(f -> expectedMessages.put(f.line, f.get(MESSAGE)));
 
     Multimap<Integer, String> missing = Multimaps.filterEntries(expectedMessages, e -> !actualMessages.containsEntry(e.getKey(), e.getValue()));
-    assertThat(missing.asMap())
+    flowAssertions.assertThat(missing.asMap())
       .overridingErrorMessage("Wrong messages in flow %s\n expected: %s\n actual: %s", flowId, expectedMessages, actualMessages)
       .isEmpty();
   }
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a call to 'assertAll' after all 'assertThat'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2970)